exercise/templatetags/exercise.py

```diff
@@ -118,6 +134,45 @@ def submission_status(status):
     return Submission.STATUS[status]
 
 
+"""
+    Used for adding relevant classes to the
+    points_badge to enable textual status.
+"""
+@register.simple_tag(takes_context=True)
+def get_points_class(context):
+    html_class = set()
+    if context['difficulty'] == "-" and context['max'] <= 2:
+        html_class.add("aplus-points")
+        class_name = ""
+
+        if ((context['max'] == 2 and context['points'] == 2)
+        or context['max'] == 1 and context['points']== 1):
+            class_name = "accepted"
+        elif (context['max'] == 2 and context['points'] == 1):
+            class_name = "ok"
+        elif context['points'] == 0 and context['submitted']:
+            class_name = "rejected"
+
+        html_class.add(class_name)
+        context['class'] = class_name
```

Comment:
This is acceptable for this case, but it's really bad as general solution. This requires order of operations. Better solution would include e.g. having single helper function for both of this template tags. Helper can use context as cache, but that is not required.

Reply:
So, should I change this part or is it good as it's "acceptable for this case"?

Reply:
It can be refactored when needed, so it's ok for now.